app.js

```diff
@@ -19,12 +20,16 @@ app.get("/api", (req,res) => {
 
 app.get("/api/topics", getTopics);
 
+app.get("/api/articles/:article_id", getArticleByID);
+
 app.all('/*', (req,res) => {
     res.status(404).send({msg: "path not found"})
 })
 
 app.use((err,req,res,next) => {
-    
+    if(err.code === '22P02') {
+        res.status(400).send({msg: 'Bad Request'})
+    }
     res.status(err.status).send({msg: err.msg})
```

Comment:
nice error handler, make sure you the final error handler in this block is that "catch all" error of 500 internal server error